setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = setuptools_golang
-version = 2.7.0
+version = 3.0.0
```

Comment:
don't touch version numbers in things you don't own

Reply:
reverted apologies